docs/services/vision/mlmodel.md

```diff
@@ -16,27 +16,31 @@ aliases:
 
 _Changed in [RDK v0.2.36 and API v0.1.118](/appendix/changelog/#vision-service)_
 
-The `mlmodel` vision service model is a machine learning detector or classifier that draws bounding boxes or returns class labels according to the specified TensorFlow Lite, TensorFlow, PyTorch, or ONNX model file available on the machine’s hard drive.
-To create a `mlmodel` detector or classifier, you need an [ML model service with a suitable model](/services/ml/).
+The `mlmodel` {{< glossary_tooltip term_id="model" text="model" >}} of the Viam vision service supports machine learning detectors and classifiers that draw bounding boxes or return class labels based on a deployed TensorFlow Lite, TensorFlow, PyTorch, or ONNX ML model.
+
+## Prerequisites
+
 Before configuring your `mlmodel` detector or classifier, you need to:
 
 {{< cards >}}
 {{% manualcard %}}
 
-<h4>Train or upload an ML model</h4>
+<h4>1. Train or upload an ML model</h4>
 
 You can [add an existing model](/services/ml/upload-model/) or [train your own models](/services/ml/train-model/) for object detection and classification using data from the [data management service](/services/data/).
```

Comment:
```suggestion
To create a `mlmodel` detector or classifier, you need an ML model service with a suitable model.

You can [add an existing model](/services/ml/upload-model/) or [train your own models](/services/ml/train-model/) for object detection and classification using data from the [data management service](/services/data/).
```
How about this?

Reply:
No I don't think that really makes sense on the card